### PR TITLE
Argo-stage index updates

### PR DESCRIPTION
**Pull request by ndushay**
I noticed some discrepancies between argo3_prod and argo_stage indexes.  This addresses those differences.

### CODE REVIEW

**Comment by ndushay** [comment]
tagging  @sul-dlss/devops

**Comment by ndushay** [comment]
@jcoyne - I asked @jmartin-sul to review this and he said:

>hmm, might be best to ask jcoyne, it's been forever since i've thought about argo solr stuff, and i'm not much of a solr expert.  at first glance, the added field names remind me of the field names we got rid of in the 2015/2016 argo solr schema overhaul, but if those fields are there and working on prod, prob best to have them in the stage schema also?

I'm always happy to remove cruft ...

File: argo3_prod/schema.xml [hunk 1]
```diff
@@ -156,8 +156,6 @@
          unknown fields indexed and/or stored by default -->
     <!--dynamicField name="*" type="ignored" multiValued="true" /-->
 
-    <!-- END hydra deprecated items -->
-
```
Comment (ndushay) [inline argo3_prod/schema.xml:160]: this is a vestigial comment -- there is no corresponding begin comment.

File: argo_stage/schema.xml [hunk 2]
```diff
@@ -155,8 +155,44 @@
...
+  <copyField source="*_dt"    dest="date" />
+  <copyField source="*_t"     dest="text" />
+  <copyField source="*_t"     dest="text_search" />
+  <copyField source="*_t"     dest="text_unstem_search" />
+  <copyField source="title_t" dest="title_t_ws" />
+  <copyField source="title_t" dest="title_t_unstem_search" />
+  <copyField source="identityMetadata_t" dest="identityMetadata_t_ws" />
+  <copyField source="identityMetadata_t" dest="identityMetadata_t_unstem_search" />
+<!-- END argo customization artifacts -->
+
```
Comment (ndushay) [inline argo_stage/schema.xml:195]: in prod, but not stage.

Comment (jcoyne) [reply argo_stage/schema.xml:195]: We no longer use these fields.  We should remove this stuff from production rather than add it to stage.

**Comment by ndushay** [comment]
@sul-dlss/devops this only changes non-operative stuff (like whitespace and comments) and therefore can be deployed to sul-solr whenever.

**Comment by ndushay** [comment]
image

image

**Comment by ndushay** [comment]
@sul-devops-team

**Comment by ndushay** [comment]
@sul-devops-team -- this only changes non-operative stuff (like whitespace and comments) and therefore can be deployed to sul-solr whenever.  It should not be forgotten tho!  :-)

**Pull request merged**